src/applications/vaos/actions/newAppointment.js

```diff
@@ -50,6 +59,46 @@ export function updateFormData(page, uiSchema, data) {
   };
 }
 
+export function updateHasCCEnabledSystems(hasCCEnabledSystems) {
+  return {
+    type: FORM_VA_SYSTEM_UPDATE_HAS_CC_ENABLED_SYSTEMS,
+    hasCCEnabledSystems,
+  };
+}
+
+export function getUserSystems() {
```

Comment:
It looks like this is only used in one place, is this function and the new actions necessary now?